api/cluster-resources/README.md

```diff
@@ -51,7 +51,45 @@ Jupyter UI extension for playground mode. Passed via GCS at cluster creation tim
 Tweak the above instructions for testing the user script to push a modified
 extension and modify the cluster controller to use it.
 
+# snippets-menu
+
+AoU Clusters enable the [Snippets Menu extension](https://jupyter-contrib-nbextensions.readthedocs.io/en/latest/nbextensions/snippets_menu/readme.html)
+with custom AoU-specific code snippets. The snippets live in
+https://github.com/all-of-us/workbench-snippets. AoU configures Leo clusters as
+follows to enable this:
+
+1. Enable the snippets_menu/main extension in ./setup_notebook_cluster.sh
+1. Deploy a Jupyter UI extension to configure the menu with AoU-specific snippets
+
+## Updating Snippets
+
+As code snippets are updated in the source repository, we will want to
+periocially update the menus in the Workbench. For now, this process is fairly
```

Comment:
How often do we expect these snippets to change / be updated? We may benefit from fully automating this into our build process sooner than later if it's going to be reasonably often, so the only thing we'd have to do is update a submodule reference.

Maybe file a tracking ticket with some loose notes on a feasible approach?

Reply:
Fair point - I don't think we know yet how frequent this will be. A submodule is a possible solution, though it is a bit more opaque than having the explicit menu JSON checked into source. I put some thoughts on https://precisionmedicineinitiative.atlassian.net/browse/RW-2665 and noted here.